deployments/audience_engagement.yaml

```diff
@@ -2,27 +2,32 @@ status: Converted
 registry_authors:
 - Nicolas Berrios
 - Jack Fitzsimons
+- James Honaker
 tier: 1
 deployment:
   name: Audience Engagement API
   data_curator: LinkedIn
   intended_use: Statistics API
   data_product_type: Summary statistics # TODO: Is this correct?
   data_product_region: Global
-  description: With these proposed parameter values (...) and applying
-    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee.
+  description: 'With these proposed parameter values (...) and applying
+    Theorem 2, we get a ﬁnal (34.9, 7 × 10−9)-DP monthly guarantee.'
   publication_date: '2020-01-01'
   additional_information: https://arxiv.org/abs/2002.05839
-  # dp_flavor: TODO
+  dp_flavor:
+    name: Approximate DP
+    data_domain: '' # TODO
+    unprotected_quantities: '' # TODO
```

Comment:
When 
- #81
is resolved these may go away, but I shouldn't block.

Reply:
will remove when that goes through